### PR TITLE
fix incorrect comparison message in api validation

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

**What this PR does / why we need it**:
This PR fixes incorrect error message in api validation.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
NONE

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by allencloud** [comment]
/assign @smarterclayton
Could you help to review this?

**Comment by allencloud** [comment]
ping @jianhuiz @janetkuo 
Could you help to review this? Thanks a lot.

**Review by janetkuo — APPROVED** [review]
This lgtm if the validation code (if statements) is correct. Otherwise we need to fix the validation code instead.

File: pkg/apis/autoscaling/validation/validation.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func validateResourceSource(src *autoscaling.ResourceMetricSource, fldPath *fiel
 	}
 
 	if src.TargetAverageUtilization != nil && *src.TargetAverageUtilization < 1 {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("targetAverageUtilization"), src.TargetAverageUtilization, "must be greater than 0"))
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("targetAverageUtilization"), src.TargetAverageUtilization, "must be greater than or equal to 1"))
```
Comment (janetkuo) [inline pkg/apis/autoscaling/validation/validation.go]: @kubernetes/sig-autoscaling-bugs

**Comment by jianhuiz** [comment]
What's the difference between `>0` and `>=1`? Any backgrounds?

**Comment by janetkuo** [comment]
> What's the difference between `>0` and `>=1`? Any backgrounds?

@jianhuiz you're right, these fields are all integers, so they mean the same thing

**Comment by janetkuo** [comment]
@allencloud can we close this one?

**Comment by allencloud** [comment]
Yes. I am afraid we can close this. It seems that I misunderstood this. 
Thanks a lot for your review @jianhuiz @janetkuo

**Pull request closed without merging**